src/pages/developers/tutorials.js

```diff
@@ -213,6 +213,30 @@ const published = (locale, published) => {
   ) : null
 }
 
+const Skill = {
+  BEGINNER: "beginner",
+  INTERMEDIATE: "intermediate",
+  ADVANCED: "advanced",
+}
+
+const SkillTranslationId = {
+  BEGINNER: "page-tutorial-beginner",
+  INTERMEDIATE: "page-tutorial-intermediate",
+  ADVANCED: "page-tutorial-advanced",
+}
+
+const getSkillTranslationId = (skill) => {
+  const lowercasedSkill = skill.toLowerCase()
+
+  if (Skill.BEGINNER === lowercasedSkill) return SkillTranslationId.BEGINNER
+  if (Skill.INTERMEDIATE === lowercasedSkill)
+    return SkillTranslationId.INTERMEDIATE
+  if (Skill.ADVANCED === lowercasedSkill) return SkillTranslationId.ADVANCED
+
+  // Returning the original string to preserve translations until we are ready to remove them.
+  return skill
+}
```

Comment:
@minimalsm If I am correct, this results in the same logic, simplified a bit:
```suggestion
const getSkillTranslationId = (skill) => {
  if (Object.values(Skill).includes(skill.toLowerCase()))
    return `page-tutorial-${lowercasedSkill}`
  // Returning the original string to preserve translations until we are ready to remove them.
  return skill
}
```

Reply:
This could shorten it even further :-)
```suggestion
const getSkillTranslationId = (skill) =>
  Object.values(Skill).includes(skill.toLowerCase())
    ? `page-tutorial-${lowercasedSkill}`
    : skill
```

Reply:
> This could shorten it even further :-)

I think shortening it this much makes it a little obscure/hard to read 🤔 What do you think?

Reply:
Depends if you're used to ternary expressions, implicit arrow function returns and boolean logic I suppose =)
Not a hill I'm dying on, but would just argue the original setup has a fair amount of redundancy